packages/rum/test/specs/apm-base.spec.js

```diff
@@ -52,10 +52,11 @@ describe('ApmBase', function() {
 
     apmBase.setInitialPageLoadName('new page load')
     apmBase.observe(TRANSACTION_END, endedTr => {
+      expect(endedTr).toEqual(tr)
```

Comment:
nit: `toEqual` only checks the properties but not if they are actually the same instance. In this case it's not going to matter that much though.

Reply:
It checks deep equals which should check most of the transaction properties so its good anyways.